_docker/drupal/drupal-filesystem/web/themes/custom/rhdp2/rhd-frontend/src/styles/rhd-theme/pages/_searchapp.scss

```
@@ -0,0 +1,270 @@
+}
+
+rhdp-search-filters {
+  margin-bottom: 1rem;
+  display: block;
+
+  // Active filter labels (not checkboxes)
+  .activeFilters {
+    rhdp-search-filter-item[active] {
+      display: inline-block;
+
+      .pf-c-label {
+        cursor: pointer;
+        margin: .25rem;
+      }
+    }
+  }
+
+  .title {
+    background-color: var(--pf-global--palette--black-100, #fafafa);
+    color: var(--rhd-theme--color-text, #151515);
+    font-family: RedHatDisplay;
+    font-size: 16px;
+    font-weight: 700;
+    padding: 1rem;
+  }
+
+  .control {
+    border: 1px solid var(--rhd-theme--color--ui-border, #ededed);
+  }
+
+  &[type="modal"] {
+    display: none;
+  }
+
+  // Mobile styles
+  @media only screen and (max-width: $pf-global--breakpoint--md) {
+
+    // Prevent rhdp-search-filters from showing below the footer.
+    flex: none;
+    border: none;
+
+    .mobile {
+      display: block;
+    }
+
+    .hide-on-mobile,
+    .desktop {
+      display: none;
+    }
+
+    .control {
+      display: flex;
+      flex-direction: column;
+      width: 100%;
+      height: 100%;
+      padding-top: 51px;
+      background: rgba(0, 0, 0, 0.5);
+      border: none;
+      z-index: 99;
+      right: 100%;
+      position: absolute;
+      top: 100px;
+    }
+
+    &[type="modal"] {
+      display: block;
+
+      .modal {
+        transition: 0.5s ease-in-out;
+        transform: translateX(100%);
+      }
+
+      .cover {
+        display: flex;
+        flex-direction: column;
+        width: 100%;
+        height: 100%;
+        padding-top: 100px;
+        background: rgba(0, 0, 0, 0.5);
+        border: none;
+        z-index: 999;
+        right: 100%;
+        position: absolute;
+        top: 0px;
+      }
+
+      .groups {
+        background: var(--rhd-theme--component-background--color, #fff);
+        flex: 1 1 100%;
+        order: 1;
+        overflow: auto;
+      }
+
+      .pf-c-check__label {
+        padding: .25rem;
+      }
+
+      .footer {
+        display: block;
+        flex: 1 0 auto;
+        height: 70px;
+        order: 2;
+        text-align: center;
+        background-color: var(--rhd-theme--ui-background-dark, #000);
+        padding: 1em;
+
+        .pf-c-button {
+          display: inline-block;
+          width: auto;
+        }
+      }
+
+      .applyFilters {
+        text-transform: uppercase;
+        background: var(--rhd-theme--color--ui-accent-dark, #c00);
+        font-weight: 600;
+        padding: 10px 25px;
+        transition: background 0.2s ease-in 0s;
+        color: var(--rhd-theme--component-background--color, #fff);
+      }
+
+      rhdp-search-filter-group {
+        margin: 0 0 0.5em 0;
+      }
+    }
+  }
+}
+
+rhdp-search-filter-group {
+  display: block;
+  margin-top: 1rem;
+  padding: 0 1rem 1rem;
+  position: relative;
+
+  .hide {
+    display: none;
+  }
+
+  .group-name {
+    color: var(--rhd-theme--color-text--subtle, #72767b);
+    font-family: RedHatDisplay;
```

```suggestion
    font-family: 'Red Hat Display', sans-serif;
```

`sans-serif` used as a fallback